packages/hdwallet-native/src/crypto/isolation/adapters/bitcoin.ts

```diff
@@ -22,6 +27,8 @@ export class ECPairAdapter implements SignerAsync, ECPairInterfaceAsync {
     this._isolatedKey = isolatedKey;
     this._publicKey = publicKey;
     this._network = network;
+    // instantiation of ecc lib required for taproot sends https://github.com/bitcoinjs/bitcoinjs-lib/issues/1889#issuecomment-1443792692
+    initEccLib(ecc as unknown as TinySecp256k1Interface);
```

Comment:
Note: as mentioned in https://github.com/shapeshift/hdwallet/pull/694/files?w=1#r1849990875, we use `@bitcoinerlab/secp256k1` instead of regular tiny-secp256k1 from bitcoinjs-lib which requires wasm support in web, but even then has one runtime error remaining still, will try to tackle in this PR, but will most likely be for a follow-up